### PR TITLE
Extensions: Fetch data from the WP Super Cache REST API

### PR DESCRIPTION
**Pull request by donnapep**
This PR eliminates some of the hard-coded data and instead fetches real data from the REST API. Field names were updated to match those returned by the API.

There are still some fields that are not being returned, so these will need to be removed in a future PR as the API changes.

### CODE REVIEW

File: client/extensions/wp-super-cache/directly-cached-files.jsx [hunk 1]
```diff
@@ -40,40 +40,40 @@ const DirectlyCachedFiles = ( {
...
+						args: { cache_path: cache_path },
 						components: { strong: <strong /> },
 					}
 				) }
 				</p>
 				}
 				<p>
 					{ translate(
-						'Directly cached files are files created directly off %(wp_cache_path)s where your blog lives. This ' +
+						'Directly cached files are files created directly off %(cache_path)s where your blog lives. This ' +
```
Comment (a8ci18n) [inline client/extensions/wp-super-cache/directly-cached-files.jsx]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Directly cached files are files created directly off %(wp_cache_path)s where your blog lives. This feature is only useful if you are expecting a major Digg or Slashdot level of traffic to one post or page.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (yoavf) [inline client/extensions/wp-super-cache/directly-cached-files.jsx]: a8ci18n shouldn't suggest the previous version of the string when it is being changed. Probably confused by the multi-line concatenation here

File: client/extensions/wp-super-cache/directly-cached-files.jsx [hunk 2]
```diff
@@ -65,40 +65,40 @@ class DirectlyCachedFiles extends Component {
...
+							args: { cache_path: cache_path },
 							components: { strong: <strong /> },
 						}
 					) }
 					</p>
 					}
 					<p>
 						{ translate(
-							'Directly cached files are files created directly off %(wp_cache_path)s where your blog lives. This ' +
+							'Directly cached files are files created directly off %(cache_path)s where your blog lives. This ' +
```
Comment (a8ci18n) [inline client/extensions/wp-super-cache/directly-cached-files.jsx]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Directly cached files are files created directly off %(wp_cache_path)s where your blog lives. This feature is only useful if you are expecting a major Digg or Slashdot level of traffic to one post or page.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by donnapep** [comment]
@dbtlr We should probably do a full sweep once we've wrapped up to make sure everything is saving and fetching correctly.

**Pull request merged**